question_answering/question_answering.py

```diff
@@ -488,13 +528,12 @@ class QuestionHandler:
     class ConfigKeys:
         pass
 
-    def __init__(self, qustion_config: dict = None):
-        if qustion_config is None:
-            qustion_config = {}
-        self.type = qustion_config.get("type") or "default"
+    def __init__(self, **kwargs):
+        self.type = kwargs.get("type", "default")
```

Comment:
no need for a type to be fed into the question handlers. They know their own type.